src/main/resources/connectionParameters.properties

```diff
@@ -13,4 +9,8 @@ Project-ID=cdf-athena
 Data-Set=test_automation
 tableDemo=DemoCheck1
 delimiter@=@
+# GCP project Id
+projectId=cdf-athena
```

Comment:
why do we still need these properties here?

Reply:
We don't need this. Moved this to `# properties to be removed`.

I will remove this properties along with  all deprecated methods in new PR.

Reply:
Okay